qemu/tests/cfg/expose_host_mtu.cfg

```diff
@@ -0,0 +1,13 @@
+- expose_host_mtu: install setup image_copy unattended_install.cdrom
+    no RHEL.6 RHEL.5 RHEL.4 RHEL.3
+    requires_root = yes
+    type = expose_host_mtu
+    kill_vm = yes
+    only virtio_net
+    start_vm = no
+    nic_extra_params = ",host_mtu=4000"
+    nm_stop_cmd = systemctl stop NetworkManager
+    check_guest_mtu_cmd = "ifconfig %s"
+    bridge_get_if_cmd = "bridge link show | awk '{print $2}'"
+    Windows: # TBD, will support later #
```

Comment:
suggest to add `no Windows`  and add some comments for now, since this script can not be tested with windows guest now indeed.

Reply:
Right, others will get confused if test fail with Windows, so please `no Windows` for not support currently. Thanks.

Reply:
Already add "no Windows" in the cfg file. Thanks.